.cirrus.yml

```diff
@@ -302,6 +302,23 @@ task:
       script:
         - ./dev/bots/firebase_testlab.sh
 
+    - name: gold_tryjob_status # linux- pre-submit- only
+      # Gold Tryjobs are only generated by pre-submit tests from contributors with the correct
+      # permissions. Those who do not have these permissions granted follow am different golden
+      # file workflow laid out in:
+      # https://github.com/flutter/flutter/wiki/Writing-a-golden-file-test-for-package:flutter
+      only_if: $CIRRUS_BRANCH != 'master' && ($CIRRUS_USER_PERMISSION == 'admin' || $CIRRUS_USER_PERMISSION == 'write')
```

Comment:
@dnfield WDYT? I don't want to run this check outside of pre-submit, or for contributors that aren't actually generating tryjobs due to permission restrictions.

Reply:
Is this something we really only want run for the autoroller?  Would it make sense to restricting to the engine-flutter-autoroll account?

Reply:
I thought maybe so, but I think it also will be really beneficial to other contributors from a time standpoint.
If a golden file change is made, you might have to re-run as many as 6 test shards after they run the first time, fail, and the change gets triaged. That could take over an hour to re-run those tests. This status check would only take a minute or two to update and be done.

Reply:
Ahh ok, that makes sense.  This line LGTM then